scheduler/common/cluster-client/src/main/kotlin/ac/uk/ebi/cluster/client/model/Job.kt

```diff
@@ -2,7 +2,9 @@ package ac.uk.ebi.cluster.client.model
 
 import ac.uk.ebi.cluster.client.lsf.LOGS_PATH
 
-data class Job(val id: String, val queue: String)
-
-val Job.logsPath: String
-    get() = "$LOGS_PATH${id}_OUT"
+data class Job(
+    val id: String,
+    val queue: String,
+) {
+    val logsPath: String = "$LOGS_PATH${id}_OUT"
```

Comment:
I agree with the configurable logs path but I don't think it's a good choice to have the queue set per configuration. I think at the moment of triggering each job, we should be able to set the queue instead of having a client per queue

Reply:
yes agree having it per job can be super usefull

Reply:
this is still hardcode